src/defs/authenticatorConfigInfoRepresentation.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * https://www.keycloak.org/docs-api/11.0/rest-api/index.html#_authenticatorconfiginforepresentation
+ */
+export default interface AuthenticatorConfigInfoRepresentation {
+  name?: string;
+  providerId?: string;
+  helpText?: string;
+  properties?: ConfigPropertyRepresentation[];
+}
+
+export interface ConfigPropertyRepresentation {
+  name?: string;
+  label?: string;
+  helpText?: string;
+  type?: string;
+  defaultValue?: any;
```

Comment:
Why is the type of `defaultValue` not known? Perhaps `object` might be a better type?

Reply:
the java type is `Object` so it can be anything if we declare it to be `object` then it will be to restrictive as it can be anything